source/includes/extracts-driver-compatibility-matrix.yaml

```diff
@@ -179,9 +179,9 @@ inherit:
 replacement:
   driver: MongoDB PHP driver
 post: |
-   In the table below, `mongodb <https://pecl.php.net/package/mongodb>`_ and
-   `PHPLIB <https://github.com/mongodb/mongo-php-library>`_ refer to the
-   {{driver}} and userland library, respectively.
+   In the table below, `ext <https://pecl.php.net/package/mongodb>`_ and
```

Comment:
Noting that the PECL website was quite slow, perhaps you want to use https://github.com/mongodb/mongo-php-driver here instead? That would be consistent with linking to the library's GitHub repo as well.

FYI: PECL is really the package manager entry. The equilvalent library link is https://packagist.org/packages/mongodb/mongodb; however, I don't think we need to point to that, since it mainly presents data extracted from the GitHub repository.

Reply:
Updated this to point to github